lib/swagcov/install.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Swagcov
+  class Install
+    def generate_dotfile
+      dotfile = Rails.root.join(".swagcov.yml").to_s
+
+      return if File.exist?(dotfile)
+
+      File.write(
+        Rails.root.join(dotfile).to_s,
+        {
+          "docs" => {
+            "paths" => ["TODO: *required* list your openapi swagger file(s) here"]
```

Comment:
I think it would actually be OK to put a default here (the default rswag file path), and leave the only/ignore options as empty arrays/strings (whichever syntax would be compatible with working out of the box).

That would leave the default as "it just works"

If that makes sense as a default, what do you think about a link back to either a wiki page or a header in the README (via a yaml comment) instead of the inline TODO?

Reply:
Perhaps a generic path with something like `your_swagger_file_here.yml`? Not everyone uses rswag so I wouldn't want to customize it to rswag.